flow/connectors/bigquery/qrep_object_pull.go

```diff
@@ -339,14 +339,16 @@ func (c *BigQueryConnector) ExportTxSnapshot(
 		if err != nil {
 			return nil, nil, fmt.Errorf("failed to start export job for table %s: %w", tm.SourceTableIdentifier, err)
 		}
-		jobs = append(jobs, job)
+		jobs[tm.SourceTableIdentifier] = job
```

Comment:
Is there any problem with running too many extract jobs at once? Should it be tied to parallelism